xl_auth/translations/sv/LC_MESSAGES/messages.po

```diff
@@ -941,110 +954,136 @@ msgstr "(Om du inte accepterar villkoren så måste du logga ut igen, knappen fi
+#: xl_auth/templates/users/inspect.html:11 xl_auth/templates/users/inspect.html:178
+#: xl_auth/templates/users/view.html:11
 msgid "Last Login At"
 msgstr "Inloggad"
 
-#: xl_auth/templates/users/home.html:43 xl_auth/templates/users/home.html:98
+#: xl_auth/templates/users/home.html:52 xl_auth/templates/users/home.html:116
 msgid "Inspect"
 msgstr "Inspektera"
 
-#: xl_auth/templates/users/home.html:47 xl_auth/templates/users/home.html:102
+#: xl_auth/templates/users/home.html:56 xl_auth/templates/users/home.html:120
 msgid "Edit Details"
 msgstr "Ändra profil"
 
-#: xl_auth/templates/users/home.html:67
-msgid "Inactive Users"
-msgstr "Inaktiva användare"
+#: xl_auth/templates/users/inspect.html:14 xl_auth/user/forms.py:20
+msgid "ToS Approved"
+msgstr "Tjänstevillkor godkända"
 
-#: xl_auth/templates/users/inspect.html:14 xl_auth/templates/users/simple_view.html:11
+#: xl_auth/templates/users/inspect.html:17 xl_auth/templates/users/simple_view.html:11
 #: xl_auth/templates/users/view.html:30
 msgid "Active Account"
 msgstr "Konto aktivt"
 
-#: xl_auth/templates/users/inspect.html:17 xl_auth/templates/users/view.html:33
+#: xl_auth/templates/users/inspect.html:20 xl_auth/templates/users/view.html:33
 #: xl_auth/user/forms.py:142
 msgid "System Administrator"
 msgstr "Systemadministratör"
 
-#: xl_auth/templates/users/inspect.html:23
+#: xl_auth/templates/users/inspect.html:26
 msgid "Users Created"
 msgstr "Användare skapade"
 
-#: xl_auth/templates/users/inspect.html:26
+#: xl_auth/templates/users/inspect.html:29
 msgid "Users Modified"
 msgstr "Användare ändrade"
 
-#: xl_auth/templates/users/inspect.html:29
+#: xl_auth/templates/users/inspect.html:32
 msgid "Permissions Created"
 msgstr "Behörigheter skapade"
 
-#: xl_auth/templates/users/inspect.html:32
+#: xl_auth/templates/users/inspect.html:35
 msgid "Permissions Modified"
 msgstr "Behörigheter ändrade"
 
-#: xl_auth/templates/users/inspect.html:35
+#: xl_auth/templates/users/inspect.html:38
 msgid "Collections Created"
 msgstr "Sigler skapade"
 
-#: xl_auth/templates/users/inspect.html:38
+#: xl_auth/templates/users/inspect.html:41
 msgid "Collections Modified"
 msgstr "Sigler ändrade"
 
-#: xl_auth/templates/users/inspect.html:41
+#: xl_auth/templates/users/inspect.html:44
 msgid "Clients Created"
 msgstr "OAuth2-klienter skapade"
 
-#: xl_auth/templates/users/inspect.html:44
+#: xl_auth/templates/users/inspect.html:47
 msgid "Clients Modified"
 msgstr "OAuth2-klienter ändrade"
 
-#: xl_auth/templates/users/inspect.html:75 xl_auth/templates/users/view.html:73
+#: xl_auth/templates/users/inspect.html:69
+msgid "Own Permissions"
+msgstr "Egna behörigheter"
+
+#: xl_auth/templates/users/inspect.html:78 xl_auth/templates/users/view.html:73
 msgid "Active Collection"
 msgstr "Sigel aktivt"
 
-#: xl_auth/templates/users/inspect.html:111
+#: xl_auth/templates/users/inspect.html:116
+msgid "Permissions Created or Modified for Other Users"
```

Comment:
What do you think of "Permissions created or modified by this user"/"Behörigheter skapade/ändrade av denna användare"? Or do we filter out permissions for the particular user in this list? That is, if I've given myself cataloging permissions, will that show up in the list?

Reply:
> /.../ That is, if I've given myself cataloging permissions, will that show up in the list?

Correct.

* [ ] Adjust wording